solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/UI/Components/Buttons/Views/PrimaryButton.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  PrimaryButton.swift
+//  SpaceApp
+//
+//  Created by Pedro Gabriel on 16/03/22.
+//
+
+import SwiftUI
+
+struct PrimaryButton: View {
+    var title: String
+    var action: (() -> Void)
+
+    init(_ title: String, action: @escaping (() -> Void)) {
+        self.title = title
+        self.action = action
+    }
+
+    var body: some View {
+        Button(title, action: action)
+            .frame(height: 60)
```

Comment:
**[SUGGESTION]**
Since button size is a common thing, consider extracting it to the DS, a constant in the button namespace or some common place.